namer/metadataapi.py

```diff
@@ -17,13 +18,19 @@
 
 import rapidfuzz
 import requests
+import requests_cache
 from loguru import logger
 from PIL import Image
 from unidecode import unidecode
 
 from namer.fileutils import make_command, set_permissions
 from namer.types import ComparisonResult, default_config, FileNameParts, LookedUpFileInfo, NamerConfig, Performer
 
+config = default_config()
+if config.enabled_requests_cache:
+    cache_file = Path(tempfile.gettempdir()) / 'namer_cache'
+    requests_cache.install_cache(str(cache_file), expire_after=timedelta(minutes=config.requests_cache_expire_minutes))
+
```

Comment:
This is why the tests are failing.   The Command object passes in a config object.   This should probably be triggered when the watchdog service is started, or if you want it to work with the command line as well, then when the config object is instantiated.